man/man4/zfs.4

```diff
@@ -501,6 +501,18 @@ For testing, pause RAID-Z expansion when reflow amount reaches this value.
 .It Sy raidz_io_aggregate_rows Ns = Ns Sy 4 Pq ulong
 For expanded RAID-Z, aggregate reads that have more rows than this.
 .
+.It Sy raidz_read_sit_out_secs Ns = Ns Sy 600 Ns s Po 10 min Pc Pq ulong
```

Comment:
All references to 'raidz' should just renamed to 'raid', since this works with dRAID (and to match the `raid_read_sit_out_secs` name in the commit message)

Reply:
It was originally, I asked Don to rename it "raidz" in this https://github.com/openzfs/zfs/pull/16900#discussion_r1902265688 so it's consistent with some other existing names.  Perhaps the best thing to do would be rename it `read_sit_out_secs`.  Then we could also generically apply it to mirrors is someday support is added for that.

Reply:
Or even, `sit_out_secs` to be consistent with the other renames.